src/Nethermind/Nethermind.Evm/Tracing/ParityStyle/ParityLikeTxTracer.cs

```diff
@@ -423,17 +423,19 @@ public void ReportStorageRead(StorageCell storageCell)
 
         public void ReportAction(long gas, UInt256 value, Address @from, Address to, ReadOnlyMemory<byte> input, ExecutionType callType, bool isPrecompileCall = false)
         {
-            ParityTraceAction action = new();
-            action.IsPrecompiled = isPrecompileCall;
-            action.IncludeInTrace = !isPrecompileCall || callType == ExecutionType.Transaction;
-            action.From = @from;
-            action.To = to;
-            action.Value = value;
-            action.Input = input.ToArray();
-            action.Gas = gas;
-            action.CallType = GetCallType(callType);
-            action.Type = GetActionType(callType);
-            action.CreationMethod = GetCreateMethod(callType);
+            ParityTraceAction action = new ParityTraceAction
+            {
+                IsPrecompiled = isPrecompileCall,
+                IncludeInTrace = !isPrecompileCall || callType == ExecutionType.Transaction || !value.IsZero,
```

Comment:
we will never include ANY zero value actions?

Reply:
this just enforces that we will include non zero value PRECOMPILE actions 
does not affect any other action.

Reply:
Question is `!(isPrecompileCall && callType != ExecutionType.Transaction && value.IsZero)` more readable? + maybe add comment that we are IGNORING those under brackets.

Reply:
done